app/views/mail_shared/panel/_proposal_attributes.html.haml

```diff
@@ -4,6 +4,8 @@
   subtext: proposal.public_id }
 
 - proposal.fields_for_display.each_slice(2) do |fields|
+  - field_1_data = !property_to_s(fields[0][1]).empty? ? property_to_s(fields[0][1]) : '—'
```

Comment:
we may want to call `blank?` rather than `empty?` here since you cannot call `empty?` on `nil`

eg:

```
[1] pry(main)> nil.empty?
NoMethodError: undefined method `empty?' for nil:NilClass
from (pry):1:in `<main>'
[3] pry(main)> nil.blank?
=> true
```